### PR TITLE
Fix/multiple menu items

### PR DESCRIPTION
**Pull request by connoratrug**
Closes: #81 
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/integration/system tested
- [x] User documentation updated
- [x] Clean commits
- [ ] No warnings during install
- [x] Added Feature/Fix to release notes

### CODE REVIEW

File: src/exampleMenu.ts [hunk 1]
```diff
@@ -33,59 +33,17 @@ const molgenisMenu: MolgenisMenu = {
...
-              }
-            ]
-          },
-          {
-            type: 'plugin',
-            id: 'tagwizard',
-            label: 'Tag Wizard'
-          }
-        ]
-      },
```
Comment (marikaris) [inline src/exampleMenu.ts]: I think it's better to delete the import plugin, as the menu now doesn't have submenu's anymore.

Comment (connoratrug) [reply src/exampleMenu.ts]: done

File: tests/e2e/specs/test.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ module.exports = {
     browser
       .url(process.env.VUE_DEV_SERVER_URL)
       .waitForElementVisible('#app', 5000)
+      .click('.jumbotron button.btn')
+      .waitForElementVisible('.navbar', 5000)
```
Comment (fdlk) [inline tests/e2e/specs/test.js]: The e2e test finds a bug I think.
The context api is stubbed with `showCookieWall: false` but the PageComponent shows it anyways.

**Comment by fdlk** [comment]
I get a warning `warning Resolution field "chromedriver@77.0.0" is incompatible with requested version "chromedriver@^2.46.0"`

**Pull request merged**